pom.xml

```diff
@@ -166,73 +151,10 @@
       <pluginManagement>
          <plugins>
             <plugin>
-               <groupId>ch.qos.cal10n.plugins</groupId>
-               <artifactId>maven-cal10n-plugin</artifactId>
-               <version>${cal10n.plugin.version}</version>
-            </plugin>
-            <plugin>
                <groupId>org.jboss.maven.plugins</groupId>
                <artifactId>maven-jdocbook-plugin</artifactId>
                <version>${jdocbook.plugin.version}</version>
                <extensions>true</extensions>
-               <dependencies>
-                  <dependency>
```

Comment:
From my understanding, these docbooks were the previous way of generating documentation. We should now be using asciidoc for this, right?